playbooks/common/openshift-cluster/upgrades/v3_0_to_v3_1/upgrade.yml

```diff
@@ -76,7 +76,7 @@
 
   - fail:
       msg: Atomic OpenShift 3.1 packages not found
-    when: g_aos_versions.curr_version | version_compare('3.0.2.900','<') and (g_aos_versions.avail_version is none or g_aos_versions.avail_version | version_compare('3.0.2.900','<'))
+    when: deployment_type == 'atomic-openshift' and (g_aos_versions.curr_version | version_compare('3.0.2.900','<') and (g_aos_versions.avail_version is none or g_aos_versions.avail_version | version_compare('3.0.2.900','<')))
```

Comment:
We don't yet support upgrades for atomic-enterprise, so the check should be for deployment_type in ['online', 'openshift-enterprise']

There would also need to be an additional check to make sure the origin package is gte 1.1